scraper.py

```
@@ -1,11 +1,16 @@
 import re
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.safari.service import Service as SafariService
+from selenium.webdriver.edge.service import Service as EdgeService
 import time
 import random
 import logging
 from database import DatabaseManager
-from config import SCRAPER_CONFIG
+import mysql.connector
+from config import SCRAPER_CONFIG, DB_CONFIG
```

Ruff (0.8.0)

`config.SCRAPER_CONFIG` imported but unused

Remove unused import: `config.SCRAPER_CONFIG`

(F401)